bot/resources/tags/codeblock.md

```diff
@@ -1,17 +1,7 @@
-Discord has support for Markdown, which allows you to post code with full syntax highlighting. Please use these whenever you paste code, as this helps improve the legibility and makes it easier for us to help you.
+Here's how to format Python code on Discord:
 
-To do this, use the following method:
-
-\```python
+\```py
 print('Hello world!')
 \```
 
-Note:  
-• **These are backticks, not quotes.** Backticks can usually be found on the tilde key.  
-• You can also use py as the language instead of python  
-• The language must be on the first line next to the backticks with **no** space between them  
-
-This will result in the following:
-```py
-print('Hello world!')
-```
+**These are backticks, not quotes.** See [here](https://superuser.com/questions/254076/how-do-i-type-the-tick-and-backtick-characters-on-windows/254077#254077) if you can't find the backtick key.
```

Comment:
"See here" sounds awkward to me.

```suggestion
**These are backticks, not quotes.** See [this](https://superuser.com/questions/254076/how-do-i-type-the-tick-and-backtick-characters-on-windows/254077#254077) if you can't find the backtick key.
```

Reply:
"See here" sounds more natural to me than "See this" but I don't feel strongly about either.

Reply:
I also think "see here" sounds good. As a third option, something like "See this article" could also be used.

Reply:
"check out" would be a fine alternative to "see", but only if it's followed by "this". Though "see here" is fewer words overall and is still my preferred option. We could also do "read this", but some people might find that slightly abrasive.

Reply:
After thinking about this for a little while now, I’m actually starting to lean toward “check this out.” It sounds quite natural to me and the added word won’t really make much of a difference in length.

Reply:
I'm just trying to figure out how to change it to "check this out" within the GitHub UI--that suggestion works for me.